### PR TITLE
WFLY-14559: Renamed and updated WFLY-14559_Add_four_new_RESTEasy_cont…

### PR DESCRIPTION
**Pull request by ronsigal**
…ext_parameters.adoc

### CODE REVIEW

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+* {issue-base-url}/EAP7-1655[EAP7-1655]
+* {issue-base-url}/WFLY-14559[WFLY-14559]
+
+=== Related Issues:
+
+* {issue-base-url}/RESTEASY-2280[RESTEASY-2280]
+* {issue-base-url}/RESTEASY-2646[RESTEASY-2646]
+* {issue-base-url}/RESTEASY-2782[RESTEASY-2782]
+* {issue-base-url}/RESTEASY-2849[RESTEASY-2849]
+* {issue-base-url}/RESTEASY-2866[RESTEASY-2866]
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:41]: Minor nit, this should just be `https://issues.redhat.com/browse`. We migrated away from the `issue-base-url` for ease of copy/paste when reviewing.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Since the RESTEasy context parameters were first exposed in the wildfly/jaxrs management model
+(https://github.com/wildfly/wildfly-proposals/blob/main/jaxrs/WFLY-12298_Change_RESTEASY_settings.adoc),
+several new parameters have been introduced, of which the following should be exposed to the user:
+
+ * resteasy.match.cache.enabled:                       [RESTEASY-2646]
+ * resteasy.match.cache.size:                          [RESTEASY-2646]
+ * resteasy.original.webapplicationexception.behavior: [RESTEASY-2782]
+ * resteasy.patch.filter.disabled:                     [RESTEASY-2280]
+ * resteasy.patch.filter.legacy:                       [RESTEASY-2849]
+ * resteasy.proxy.implement.all.interfaces:            [RESTEASY-2866]
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:22]: Should we drop the `resteasy` prefix from the model attribute names? I know most of them have that prefix now, but it's not strictly required and saves some typing for administators.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+The wildfly/jaxrs module should be extended to expose these new parameters.
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:24]: s/wildfly/WildFly for consistency with other references.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Since the RESTEasy context parameters were first exposed in the wildfly/jaxrs management model
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:13]: s/wildfly/WildFly

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+Engineering
+
+=== Affected Projects or Components:
+
+* WildFly
+* RESTEasy
+
+== Requirements
+
+* Some simple modifications to the wildfly/jaxrs module will be necessary to add these new parameters.
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:62]: s/wildfly/WildFly

**Review by jamezp — COMMENTED** [review]
Just a few minor comments.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+The proper behavior of the existing parameters is already tested in RESTEasy, as follows:
+
+ * resteasy.match.cache.enabled, resteasy.match.cache.size:
+ 
+   resteasy-core/src/test/java/org/jboss/resteasy/core/registry/RootNodeCacheSizeTest.java   
+    
+ * resteasy.original.webapplicationexception.behavior:
+ 
+   testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/exception/ClientWebApplicationExceptionMicroProfileProxyTest.java
```
Comment (jamezp) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:75]: Should these be links? I'm not convinced they should be, but it seems like we could easily enough make them links to the source.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 7]
```diff
@@ -0,0 +1,102 @@
+= WFLY-14559 Add six new RESTEasy context parameters
```
Comment (bstansberry) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:1]: Please add the jekyll front matter (the categories stuff).

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+The wildfly/jaxrs module should be extended to expose these new parameters.
+
+[Note. RESTEasy uses '.' as a separator in parameter names. WildFly/jaxrs uses '-'. 
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-1655[EAP7-1655]
```
Comment (bstansberry) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:32]: Nit: this is a related issue; this does not directly address EAP7-1655.

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+ * resteasy.match.cache.size:                          [RESTEASY-2646]
+ * resteasy.original.webapplicationexception.behavior: [RESTEASY-2782]
+ * resteasy.patch.filter.disabled:                     [RESTEASY-2280]
+ * resteasy.patch.filter.legacy:                       [RESTEASY-2849]
+ * resteasy.proxy.implement.all.interfaces:            [RESTEASY-2866]
+
+The wildfly/jaxrs module should be extended to expose these new parameters.
+
+[Note. RESTEasy uses '.' as a separator in parameter names. WildFly/jaxrs uses '-'. 
+
```
Comment (bstansberry) [inline jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:27]: Please add the Stability section.

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc (hunk 6 above)
Comment (ronsigal) [reply jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc:75]: Done, except for the WildFly tests. I guess I could add those after WildFly is released.

**Comment by ronsigal** [comment]
I didn't realize there was new activity here. I'll get on it.

**Comment by ronsigal** [comment]
I don't know how I closed this.

**Comment by ronsigal** [comment]
Reopening

**Pull request closed without merging**